### DISCUSSION TITLE
Proxy mode only use one Stream?

### DISCUSSION THREAD

**Discussion by LuckyZone** [bluenviron/mediamtx#552] [Questions]
Description

ffmpeg -i rtsp://admin:pwd@192.168.1.12/Streaming/Channels/301?transportmode=unicast

``
Input #0, rtsp, from 'rtsp://admin:pwd@192.168.1.12/Streaming/Channels/301?transportmode=unicast':
  Metadata:
    title           : HIK Media Server V4.30.062
    comment         : HIK Media Server Session Description : standard
  Duration: N/A, start: 0.000000, bitrate: N/A
    Stream #0:0: Video: hevc (Main), yuvj420p(pc, bt709), 1920x1080 [SAR 1:1 DAR 16:9], 20 fps, 25 tbr, 90k tbn, 20 tbc
    Stream #0:1: Audio: pcm_alaw, 8000 Hz, mono, s16, 64 kb/s

This is a IP cam rtsp source,may I only proxy Stream #0:0?

**Comment by aler9** [comment]
You can do this (proxying a single track instead of two) by using ffmpeg to read the stream from the camera and publish it to the server:

```
ffmpeg -i rtsp://admin:pwd@192.168.1.12/Streaming/Channels/301?transportmode=unicast -map 0:v:0 -c copy -f rtsp rtsp://localhost:8554/stream
```

[a comment by an automated account is not shown]
